master/static/srv/notebook-entrypoint.sh

```diff
@@ -18,6 +18,30 @@ set -x
 test -f "${STARTUP_HOOK}" && source "${STARTUP_HOOK}"
 set +x
 
+if [ "$PACHD_ADDRESS" != "" ] && [ "$SKIP_PACHYDERM_INSTALL" != "true" ]; then
```

Comment:
I don't see any config for `$SKIP_PACHYDERM_INSTALL`, yet. I don't have an educated opinion about whether it should be the default and if to bundle it into environment images, but I do want to be sure that the configuration won't slip though the cracks.

I think my suggestion here is if this PR is going to be merged before the variable can be populated, that it be merged without that check, and that when we add the configuration we add it back in in that PR.

Reply:
Setting SKIP_PACHYDERM_INSTALL should not be the default and should only be set if for some reason the customer can't download the extension or pachctl. E.g. airgapped installs

Reply:
Hi @wes-turner , this was suggested by me. I don't think there's a scenario yet where we have a customer that uses MLDM + MLDE in an air-gapped environment. However, I would not be surprised if we get one within this year.

My thoughts were that we'd simply tell users to stick that environment variable in as part of their task container defaults in master.yaml/helm values, or in their config template when starting a notebook. 

I hope that makes sense, and if it doesn't I'm open to suggestion. I think this approach will be more annoying to deal with in an air-gapped environment if we merge this without a way to prevent the install (off the top of my head, we'd have to tell them to modify the static file in the master pod)